wgpu-hal/src/metal/surface.rs

```diff
@@ -31,27 +32,29 @@ extern "C" fn layer_should_inherit_contents_scale_from_window(
     YES
 }
 
-const CAML_DELEGATE_CLASS: &str = "HalManagedMetalLayerDelegate";
-static CAML_DELEGATE_REGISTER: Once = Once::new();
+static CAML_DELEGATE_REGISTER: OnceCell<&'static Class> = OnceCell::new();
```

Comment:
do I understand correctly that we just need a pointer, and that's why we use `OnceCell` here?
In this case, we should be able to avoid `once-cell` dependency and instead just declare a dummy static thing, to get a pointer from.